pkg/cmd/release/list/list.go

```diff
@@ -135,6 +135,7 @@ func listRun(cmd *cobra.Command, f factory.Factory, flags *ListFlags) error {
 
 	return output.PrintArray(allReleases, cmd, output.Mappers[ReleaseViewModel]{
 		Json: func(item ReleaseViewModel) any {
+			// TODO should the ReleaseID go in the JSON as well as the release number? How about channelID?
```

Comment:
I'd like to discuss this.

We've decided to steer away from ID's, and now the "human readable" output just says the channel name and release version string. -f basic just says the version string.

But I feel like the JSON should still contain the ID. 🔥🔥🔥??
At the moment it is halfway housed, we get the channel ID, but not the release ID

(Note this applies to both `release list` and `release create`, they output the same structure)